src/components/AnArticle.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ReactHtmlParser from 'react-html-parser';
+
+const ratings = (aRating) => {
+  const stars = [];
+  const aStar = (<FontAwesomeIcon icon="star" />);
```

Comment:
@Etomovich, don't place JSX in constant variables unless they are components you will export.

Reply:
I never knew this, thanks for the insight I will refactor

Reply:
@Etomovich, have you implemented this feedback